### PR TITLE
Lesson002

### PR DESCRIPTION
**Pull request by AnMatveeva**

### CODE REVIEW

File: lesson002/task2/styles/3style.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    font-weight: bold;
+    font-size: 80%;
+    color: white;
+    position:absolute;
+    top: 30px;
+}
+#col1, #col2, #col3 {
+    height: 200px;
+    font-family: 'Times New Roman', Times, serif;
+    font-size: 100%;
```
Comment (Onix-one) [inline lesson002/task2/styles/3style.css]: when you use numbers when naming classes, use dash.

**Review by Onix-one — CHANGES_REQUESTED** [review]

File: lesson002/task2/styles/2style.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+.content {
+    width:70%; 
+    height:700px; 
+    background-color: rgb(151, 171, 171); 
+    float:left;
+}
+.podval {
+    width:100%; 
+    height:100px; 
+    background-color: rgb(29, 18, 253);
```
Comment (Onix-one) [inline lesson002/task2/styles/2style.css]: Use the height ratio too with the help of percentages. Footer not visible on my screen

File: lesson002/task2/task2-2.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        <title></title>
+        <link href="styles/3style.css" rel="stylesheet" >
+    </head>
+    <body>
+        <div id="header1">header1</div>
+        <div id="header2">header2</div>
+        <div id="header3">header3</div>
+        <div id="col1">col1</div>
+        <div id="col2">col2</div>
+        <div id="col3">col3</div>
```
Comment (Onix-one) [inline lesson002/task2/task2-2.html]: Should be full screen. Also in percentage terms.

File: lesson002/task3/styles/4style.css [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+}
+.div2 {
+    width:100%;
+    min-height:700px;
+}
+.menu {
+    width:30%; 
+    height:700px; 
+    background-color: rgb(118, 197, 102); 
+    float:left;
```
Comment (Onix-one) [inline lesson002/task3/styles/4style.css]: The same situation. Need to use % ))))

**Review by Onix-one — CHANGES_REQUESTED** [review]

File: lesson002/task4/task4-2.html [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+                width: 350px;
+                top: 25%;
+                left: 35%;
+                bottom: 45%;
+                text-align: center;
+                background: linear-gradient(#ffffff,#d3d3d3);
+                border-radius: 1%;
+                
+            }
+            p {
```
Comment (Onix-one) [inline lesson002/task4/task4-2.html]: Styles should not be in the header

File: lesson002/task4/task4.html [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+<html leng=en">
+    <head>
+        <meta charset="utf-8" />
+        <title></title>
+        <link href="styles/5style.css" rel="stylesheet" >
+    </head>
+    <body>
+        <form>
+            <fieldset>
+            <legend><h2>Personal infomation</h2></legend>
```
Comment (Onix-one) [inline lesson002/task4/task4.html]: The form should be in the center of the screen just like the bootstrap form. The form is incorrect. Hint. In order to get the separation between the forms as in the original (line-text-line) use pseudo classes and pseudo elements :before and :after

File: lesson002/task4/task4-2.html [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+                        <input type="email" class="form-control" id="email" placeholder="Username or Email" required="" autofocus="">
+                    </div>
+                <div class="form-group col-md-12">
+                    <label for="password" class="sr-only">Password</label>
+                    <input type="password" class="form-control" id="password" placeholder="Password" required>
+                </div>
+                <label class="form-check-label">
+                    <input class="input1" type="checkbox">Remember me on this computer</label>
+                    <button type="submit" class="btn btn-outline-info">Login</button>
+                </form>
```
Comment (Onix-one) [inline lesson002/task4/task4-2.html]: The button in the original with rounded corners

**Review by Onix-one — CHANGES_REQUESTED** [review]

File: lesson002/task5/t5.html [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+        <link rel="stylesheet" href="css/maine.css">
+
+        <title>Site</title>
+    </head>
+    <body class="bg-dark">
+        <div class="container">
+        <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+            <a class="navbar-brand" href="t5.html"><i>MySite</i></a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navAlt" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
```
Comment (Onix-one) [inline lesson002/task5/t5.html]: Usually the start page is called the index.html

File: lesson002/task5/t5.html [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+                </div>
+              </div>
+              <div class="carousel-item">
+                <img src="img/1500x500 (1).jfif" class="d-block w-100" alt="banner 2">
+                <div class="carousel-caption">
+                  <h3>GitHub</h3>
+                  <p class="lead">Please see my GitHub.</p>
+                  <a class="btn btn-large btn-outline-light" href="https://github.com/AnMatveeva">My GitHub</a>
+                </div>
+              </div>
```
Comment (Onix-one) [inline lesson002/task5/t5.html]: May not notice the link to the github in the carousel, so I would put it in the body.  Think about it

File: lesson002/task5/t5.html [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+            </div>
+           <button class="carousel-control-prev bg-secondary border-0" type="button" data-target="#carouselExampleControls" data-slide="prev">
+              <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+              <span class="sr-only">Previous</span>
+            </button>
+            <button class="carousel-control-next bg-secondary border-0" type="button" data-target="#carouselExampleControls" data-slide="next">
+              <span class="carousel-control-next-icon" aria-hidden="true"></span>
+              <span class="sr-only">Next</span>
+            </button>
```
Comment (Onix-one) [inline lesson002/task5/t5.html]: Look at the carousel buttons. They step on the heder. Is that how it should be?

**Review by Onix-one — CHANGES_REQUESTED** [review]

File: lesson002/task5/css/maine.css [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+h3 {
+    text-align: left;
+    margin-left: 55px;
+}
+input {
+  margin-left: 25%;
+}
+footer .col-12, footer .col-6, .h33 {
+    text-align: center;
+}
```
Comment (Onix-one) [inline lesson002/task5/css/maine.css]: .h33      Do you like this class name yourself?

File: lesson002/task5/contacts.html [hunk 12]
```diff
@@ -0,0 +1,75 @@
...
+        <link rel="stylesheet" href="css/maine.css">
+
+        <title>Site</title>
+    </head>
+    <body class="bg-dark">
+        <div class="container">
+        <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+            <a class="navbar-brand" href="t5.html"><i>MySite</i></a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navAlt" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+              <span class="navbar-toggler-icon"></span>
```
Comment (Onix-one) [inline lesson002/task5/contacts.html]: Try to make a fixed header. This is just a suggestion

**Review by Onix-one — CHANGES_REQUESTED** [review]

File: lesson002/task2/styles/3style.css (hunk 1 above)
Comment (AnMatveeva) [reply lesson002/task2/styles/3style.css]: Done

File: lesson002/task2/styles/2style.css (hunk 2 above)
Comment (AnMatveeva) [reply lesson002/task2/styles/2style.css]: Done

File: lesson002/task2/task2-2.html (hunk 3 above)
Comment (AnMatveeva) [reply lesson002/task2/task2-2.html]: Done

File: lesson002/task3/styles/4style.css (hunk 4 above)
Comment (AnMatveeva) [reply lesson002/task3/styles/4style.css]: Done

File: lesson002/task4/task4-2.html (hunk 5 above)
Comment (AnMatveeva) [reply lesson002/task4/task4-2.html]: Done

File: lesson002/task4/task4-2.html (hunk 7 above)
Comment (AnMatveeva) [reply lesson002/task4/task4-2.html]: Done

File: lesson002/task5/t5.html (hunk 8 above)
Comment (AnMatveeva) [reply lesson002/task5/t5.html]: Done

File: lesson002/task5/t5.html (hunk 9 above)
Comment (AnMatveeva) [reply lesson002/task5/t5.html]: Done

File: lesson002/task5/t5.html (hunk 10 above)
Comment (AnMatveeva) [reply lesson002/task5/t5.html]: Done

File: lesson002/task5/css/maine.css (hunk 11 above)
Comment (AnMatveeva) [reply lesson002/task5/css/maine.css]: Done

File: lesson002/task5/contacts.html (hunk 12 above)
Comment (AnMatveeva) [reply lesson002/task5/contacts.html]: Done

File: lesson002/task4/task4.html (hunk 6 above)
Comment (AnMatveeva) [reply lesson002/task4/task4.html]: Done

**Pull request merged**